prometheus_client/multiprocess.py

```diff
@@ -90,6 +97,29 @@ def collect(self):
         return metrics.values()
 
 
+class MultiProcessMetricsHandler(BaseHTTPRequestHandler):
```

Comment:
It doesn't make sense to add this, it's too use-case specific. Allowing a reigstry to be passed into the metricshandler would make more sense.

Reply:
Once you pass registry to MetricsHandler, you still need to call `MultiProcessCollector(registry)`.

Is there a smart way to do it?

@brian-brazil Thx++ for your feedback!

Reply:
There's no smart way, it's just a line of code.